tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -1117,6 +1144,7 @@ handle_build() {
 	kernel-confidential) install_kernel_confidential ;;
 
 	kernel-dragonball-experimental) install_kernel_dragonball_experimental ;;
+
```

Comment:
Extra line added.

Reply:
Yes, otherwise we would have 
```
kernel-confidential) install_kernel_confidential ;;

kernel-dragonball-experimental) install_kernel_dragonball_experimental ;;
kernel-nvidia-gpu-dragonball-experimental) install_kernel_nvidia_gpu_dragonball_experimental ;;

kernel-nvidia-gpu) install_kernel_nvidia_gpu ;;
```